.github/workflows/ur-build-hw.yml

```diff
@@ -136,12 +136,18 @@ jobs:
       run: cmake --install ${{github.workspace}}/build
 
     - name: Test adapter specific
-      run: ctest -C ${{matrix.build_type}} --test-dir ${{github.workspace}}/build --output-on-failure -L "adapter-specific" -E "memcheck" --timeout 600
+      env:
+        ZE_ENABLE_LOADER_DEBUG_TRACE: 1
+        ZE_DEBUG: 1
+      run: ctest -C ${{matrix.build_type}} --test-dir ${{github.workspace}}/build --output-on-failure -L "adapter-specific" -E "memcheck" --timeout 600 -VV
       # Don't run adapter specific tests when building multiple adapters
       if: ${{ matrix.adapter.other_name == '' }}
 
     - name: Test adapters
-      run: env UR_CTS_ADAPTER_PLATFORM="${{matrix.adapter.platform}}" ctest -C ${{matrix.build_type}} --test-dir ${{github.workspace}}/build --output-on-failure -L "conformance" --timeout 600
+      env:
+        ZE_ENABLE_LOADER_DEBUG_TRACE: 1
+        ZE_DEBUG: 1
+      run: env UR_CTS_ADAPTER_PLATFORM="${{matrix.adapter.platform}}" ctest -C ${{matrix.build_type}} --test-dir ${{github.workspace}}/build --output-on-failure -L "conformance" --timeout 600 -VV
```

Comment:
Should we move `UR_CTS_ADAPTER_PLATFORM` to `env:` section as well?

Reply:
I think that would be best added in a separate PR to improve the settings in the UR workflows. Teh additional L0 envs were for seeing when the loader is init in case of failures.